[2024-12-03T06:58:11.133+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-03T06:58:11.151+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nyc_green_tripdata.load_tripdata_to_bq manual__2024-12-03T06:57:48.596223+00:00 [queued]>
[2024-12-03T06:58:11.162+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nyc_green_tripdata.load_tripdata_to_bq manual__2024-12-03T06:57:48.596223+00:00 [queued]>
[2024-12-03T06:58:11.162+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-12-03T06:58:11.175+0000] {taskinstance.py:2330} INFO - Executing <Task(GCSToBigQueryOperator): load_tripdata_to_bq> on 2024-12-03 06:57:48.596223+00:00
[2024-12-03T06:58:11.187+0000] {standard_task_runner.py:64} INFO - Started process 1122 to run task
[2024-12-03T06:58:11.194+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nyc_green_tripdata', 'load_tripdata_to_bq', 'manual__2024-12-03T06:57:48.596223+00:00', '--job-id', '192', '--raw', '--subdir', 'DAGS_FOLDER/dag_green_tripdata.py', '--cfg-path', '/tmp/tmpdq6i7wsb']
[2024-12-03T06:58:11.201+0000] {standard_task_runner.py:91} INFO - Job 192: Subtask load_tripdata_to_bq
[2024-12-03T06:58:11.260+0000] {task_command.py:426} INFO - Running <TaskInstance: nyc_green_tripdata.load_tripdata_to_bq manual__2024-12-03T06:57:48.596223+00:00 [running]> on host 39c047f38cb8
[2024-12-03T06:58:11.367+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='muhii.smta@gmail.com' AIRFLOW_CTX_DAG_OWNER='Muhii' AIRFLOW_CTX_DAG_ID='nyc_green_tripdata' AIRFLOW_CTX_TASK_ID='load_tripdata_to_bq' AIRFLOW_CTX_EXECUTION_DATE='2024-12-03T06:57:48.596223+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-03T06:57:48.596223+00:00'
[2024-12-03T06:58:11.370+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-03T06:58:11.373+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-12-03T06:58:11.374+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-12-03T06:58:11.376+0000] {base.py:84} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-12-03T06:58:11.377+0000] {gcs_to_bigquery.py:381} INFO - Using existing BigQuery table for storing data...
[2024-12-03T06:58:11.419+0000] {gcs_to_bigquery.py:385} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'northern-shield-437204-m0', 'datasetId': 'nyc_taxi', 'tableId': 'green_tripdata'}, 'sourceFormat': 'PARQUET', 'sourceUris': ['gs://green_taxi_bucket/transform/tf_green_tripdata_2023-02.parquet'], 'writeDisposition': 'WRITE_APPEND', 'ignoreUnknownValues': False}}
[2024-12-03T06:58:11.421+0000] {bigquery.py:1702} INFO - Inserting job ***_nyc_green_tripdata_load_tripdata_to_bq_2024_12_03T06_57_48_596223_00_00_4d815f6e204f906aa344c6686d3ef58b
[2024-12-03T06:58:12.883+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-03T06:58:12.884+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 446, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.9/site-packages/google/cloud/bigquery/job/base.py", line 971, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.9/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Provided Schema does not match Table northern-shield-437204-m0:nyc_taxi.green_tripdata. Field ehail_fee has changed type from INTEGER to FLOAT; reason: invalid, message: Provided Schema does not match Table northern-shield-437204-m0:nyc_taxi.green_tripdata. Field ehail_fee has changed type from INTEGER to FLOAT
[2024-12-03T06:58:12.901+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=nyc_green_tripdata, task_id=load_tripdata_to_bq, run_id=manual__2024-12-03T06:57:48.596223+00:00, execution_date=20241203T065748, start_date=20241203T065811, end_date=20241203T065812
[2024-12-03T06:58:12.916+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 192 for task load_tripdata_to_bq (400 Provided Schema does not match Table northern-shield-437204-m0:nyc_taxi.green_tripdata. Field ehail_fee has changed type from INTEGER to FLOAT; reason: invalid, message: Provided Schema does not match Table northern-shield-437204-m0:nyc_taxi.green_tripdata. Field ehail_fee has changed type from INTEGER to FLOAT; 1122)
[2024-12-03T06:58:12.939+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-12-03T06:58:12.973+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-03T06:58:12.975+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
